cloud/auth/auth.go

```diff
@@ -289,7 +290,7 @@ func CheckUserSession(c *config.Context, client astro.Client, coreClient astroco
 			break
 		}
 	}
-	err = c.SetOrganizationContext(activeOrg.Id, activeOrg.ShortName)
+	err = c.SetOrganizationContext(activeOrg.Id, activeOrg.ShortName, fmt.Sprintf("%s", *activeOrg.Product)) //nolint
```

Comment:
Do we need to check if `*activeOrg.Product` is nil? Or will it always have a value?